content/documentation/pages/4-batch-developer-guides/2-batch/2-spring-batch.md

```diff
@@ -8,34 +8,310 @@ description: 'Create a Spring Batch Job'
 
 In this guide we will develop a Spring Batch application and deploy it to Cloud Foundry, Kubernetes, and on your local machine. In another guide, we will deploy the Spring Batch application using Data Flow.
 
-We will start from initializr and create a Spring Batch application.
-Note for CF we need a manifest, for k8s we need a service/deployment yaml.
+We will start from [Spring Initializr](https://start.spring.io/) and create a Spring Batch application.
+
+NOTE: All code for this project can be found [here](https://github.com/spring-cloud/spring-cloud-dataflow-samples/tree/master/dataflow-website/batch-developer-guides/batch/batchsamples).
 
 ## Development
 
-Explaine what we are doing to do development wise.
+Suppose a cell phone data provider needs to create billing statements for customers. The usage data is stored in JSON files that are stored on the file system. The billing solution must pull data from these files, generate the billing data from this usage data, and store it in a `BILL_STATEMENTS` table.
+
+We could implement this entire solution into a single Spring Boot Application that utilizes Spring Batch, however for this example we will break up the solution into 2 phases:
+
+1. [billsetuptask](/documentation/batch-developer-guides/batch/simple-task): The [billsetuptask](/documentation/batch-developer-guides/batch/simple-task) application will be a Spring Boot application using Spring Cloud Task that will simply create the `BILL_STATEMENTS` table.
+1. _billrun_: The _billrun_ application will be a Spring Boot application using Spring Cloud Task and Spring Batch that will read usage data from a json file and price the each row and put the resulting data into the `BILL_STATEMENTS` table.
+
+For this section we will create a Spring Cloud Task/Spring Batch billrun application that will read usage information from a json file containing customer usage data and price each entry and place the result into the `BILL_STATEMENTS` table.
+
+![BILL_STATMENTS](images/bill_statements.png)
+
+### Intoducing Spring Batch
+
+In short Spring Batch is a lightweight, comprehensive batch framework designed to enable the development of robust batch applications. Spring Batch provides reusable functions that are essential in processing large volumes of records by offering features such as:
+
+- Logging/tracing
+- Chunk based processing
+- Declarative I/O
+- Start/Stop/Restart
+- Retry/Skip
+- Resource management
+
+It also provides more advanced technical services and features that will enable extremely high-volume and high performance batch jobs through optimization and partitioning techniques.
 
-Create a single step batch job, file item reader, file item writer, no other dependencies outside of spring batch.
+For this guide we will focus on 5 Spring Batch components:
+
+![BILL_STATMENTS](images/spring-batch-reference-model.png)
+
+- Job: A `job` is an entity that encapsulates an entire batch process. A job is comprised of one or more `steps`.
```

Comment:
Nope. Since we're referring to the concept of a job and not the class name, we can use the lower case ("job"). If we referred to the class, we'd need to use code font and capitalize it ("`Job`").